values.yaml

```diff
@@ -44,21 +44,20 @@ resources: {}
 amqp:
 #  # Set managementUrl to http://rabbitmq-rabbitmq-ha.default.svc.cluster.local:15672 if using helm rabbitmq-ha
   managementUrl:
-#  # Set if not using existingRabbitSecret
+#  # Set host, username & password if not using existingRabbitSecret
   host:
   username:
   password:
+  port: 5672
+  connectionTimeout: 20
+  sslEnabled: false
+  sslKeyStore: rabbitmq-security/keycert.p12
+  sslKeyStorePassword: changeit
```

Comment:
I'd leave these two empty:
  sslKeyStore: rabbitmq-security/keycert.p12
  sslKeyStorePassword: changeit
And I'd move sslKeyStorePassword to the secret instead of configmap